drivers/NFSSR.py

```diff
@@ -273,6 +274,7 @@ def generate_config(self, sr_uuid, vdi_uuid):
         resp['sr_uuid'] = sr_uuid
         resp['vdi_uuid'] = vdi_uuid
         resp['sr_sm_config'] = self.sr.sm_config
+        resp['sr_other_config'] = self.sr.other_config
```

Comment:
Fair enough.  The version here is better to test with patch queue applied, and it should look consistent at that point. Or you can use my working branch, which has it done automatically.